lib/index.ts

```diff
@@ -1,281 +1,4 @@
-import newDebug from 'debug';
```

Comment:
Most of `index.ts` has been moved to `policy.ts` to better control the exposed API when the library is packaged